### ISSUE TITLE
BUG: broken link to sepgsql.googlecode.com reference

### ISSUE THREAD

**Issue by b4r** [SELinuxProject/selinux-notebook#16]
On the PDF generated content page 113 it references a link to https://sepgsql.googlecode.com/files/LCA20090120-lapp-selinux.pdf

Through a quick search I found a github repo that appears to have this file although I'm not familiar with whether this should be the correct reference @ https://raw.githubusercontent.com/kaigai/slides/master/20090120_LCA2009_LAPP_SELINUX.pdf

A patch of the URL substitution is below:

```patch
diff --git a/src/apache_support.md b/src/apache_support.md
index 48c8293..f53a512 100644
--- a/src/apache_support.md
+++ b/src/apache_support.md
@@ -41,7 +41,7 @@ stack has the following support:
 and Python do have support for libselinux functions in packages: PHP - with
 the *php-pecl-selinux* package, Python - with the *libselinux-python* package.

-The "[A secure web application platform powered by SELinux](http://sepgsql.googlecode.com/files/LCA20090120-lapp-selinux.pdf)"
+The "[A secure web application platform powered by SELinux](https://raw.githubusercontent.com/kaigai/slides/master/20090120_LCA2009_LAPP_SELINUX.pdf)"
 document gives a good overview of the LAPP architecture.

 ## *mod_selinux* Overview
```

**Comment by pcmoore** [comment]
Hi @b4r, this looks like it is still a problem so I'm going to reopen this issue.  Would you mind turning your suggested patch into a PR?

**Comment by pcmoore** [comment]
Ping @b4r ... would you be able to submit a PR for this?  If not, would you mind if I turned your suggested change into a proper patch/PR?

**Comment by pcmoore** [comment]
Fixed in 1ed0ec1819ff05aa8fcfc95e95dca40c8e218185.
